extensions/amazon-lambda/runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AmazonLambdaTemplate.java

```diff
@@ -86,8 +95,9 @@ public void run() {
 
                 } catch (Exception e) {
                     try {
+                        e.printStackTrace();
```

Comment:
We really should use a logger here and a few lines below (`ex.printStackTrace();`)